configure

```diff
@@ -99,6 +99,26 @@ check_do_prerequisites()
 	doctl compute domain records list "$DOMAIN" >/dev/null 2>&1 && msg check "  DO is managing $DOMAIN" || { errmsg "  error: domain not found. Please delegate '$DOMAIN' to DO's DNS service."; exit -1; }
 }
 
+check_aws_prerequisites()
+{
+	# see if we have eksctl
+	hash eksctl 2>/dev/null && msg check "  found eksctl" || { errmsg "  error: no eksctl on path. Install eksctl."; exit -1; }
```

Comment:
Advise the user to learn more about getting `eksctl` at https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html

Reply:
Changing to `error: no eksctl on path. Learn more about installing eksctl at https://eksctl.io/`